### PR TITLE
two-new-card-achievement-section

### PR DESCRIPTION
**Pull request by Darshna-Shrivastava**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -506,6 +506,36 @@ <h3 class="name-body">
...
+              <div class="text-content">
+                <h2 class="person-name">
+                  Harsh Jain, Akask Dubey, Nishant Roy
+                </h2>
+                <h3 class="name-body">
+                  Covackathon 2021 <br> First prize in 3-D designing of the product in Autodesk <br> (<i class="fas fa-rupee-sign"></i> 50,000)
+                </h3>
+              </div>
+            </div>
+          </div>
```
Comment (Darshna-Shrivastava) [inline index.html:523]: New card has been added

**Review by ashish11chawda — CHANGES_REQUESTED** [review]
Hey @Darshna-Shrivastava 
Thanks for updating the details
In my opinion it'll be great if you add these cards at the beginning of the container as these are latest updates

**Review by Sam-Ny — COMMENTED** [review]
@Darshna-Shrivastava , Also it would be great if we can have one more card to add onto it, coz it is looking uneven . Ask @adivya-5 for more details.

**Review by ashish11chawda — APPROVED** [review]
LGTM ✅

**Pull request merged**